src/index.html

```diff
@@ -10,7 +10,49 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__content">
+    <form
+      action="#"
+      method="POST"
+      class="page__form--top search-form"
+      data-qa="big"
+    >
+      <label class="input-search">
```

Comment:
<img width="828" alt="Screenshot 2023-09-15 at 10 33 41" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/53228259/cc3b42b6-8a06-4184-8b69-c30429ade85d">

try to delete label, and use only form with image and input inside